src/test/java/com/wikia/webdriver/common/logging/VelocityWrapper.java

```diff
@@ -185,7 +185,7 @@ static String fillButton(String id, String label) {
 
   static String fillHeader(String date, String polishDate, String browser, String os,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Method has 8 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00107)